components/booking/pages/BookingPage.tsx

```diff
@@ -195,7 +200,9 @@ const BookingPage = (props: BookingPageProps) => {
     })(),
   });
 
-  const getLocationValue = (booking: Pick<BookingFormValues, "locationType" | "phone">) => {
+  const getLocationValue = (
+    booking: Pick<BookingFormValues, "locationType" | "phone" | "integrations:jitsi">
+  ) => {
```

Comment:
This is causing a type error and it seems unnecessary
```suggestion
  const getLocationValue = (booking: Pick<BookingFormValues, "locationType" | "phone">) => {
```